modules/visual_script/editor/visual_script_editor.h

```diff
@@ -196,7 +196,7 @@ class VisualScriptEditor : public ScriptEditorBase {
 	int _create_new_node_from_name(const String &p_text, const Vector2 &p_point);
 	void _selected_new_virtual_method(const String &p_text, const String &p_category, const bool p_connecting);
 
-	int error_line;
+	int error_line = 0;
```

Comment:
Based on the code using it in the `.cpp`, it seems -1 would be a better default value (that's what `set_debugger_active` resets it to).